src/App.js

```diff
@@ -1,24 +1,73 @@
-import logo from './logo.svg';
-import './App.css';
+import { Layout } from "./feature/Layout";
+import { SharedPage } from "./page-layout/SharedPage";
+import "./global.css";
+import { FolderInfo } from "./ui/FolderInfo";
+import { SearchBar } from "./ui/SearchBar";
+import { CardList } from "./ui/CardList";
+import { useGetFolder } from "data-access/useGetFolder";
+import { ReadOnlyCard } from "ui/ReadOnlyCard";
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+import { FolderPage } from "page-layout/FolderPage/FolderPage";
+import { AddLinkBar } from "ui/AddLinkBar/AddLinkBar";
+import { SharedPossibleCard } from "ui/SharedPossibleCard";
+import { getFolders } from "data-access/getFolders";
+import { useEffect, useState } from "react";
+import { FolderContent } from "ui/FolderContent/FolderContent";
 
 function App() {
+  const [folders, setFolders] = useState();
+
+  const { data } = useGetFolder();
+  const { profileImage, ownerName, folderName, links } = data || {};
+
+  const handleLoadFolders = async () => {
+    const { data } = await getFolders();
+    setFolders(data);
+  };
+
+  useEffect(() => {
+    handleLoadFolders();
+  }, []);
+
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <BrowserRouter>
+      <Layout>
+        <Routes>
+          <Route
+            path="/shared"
+            element={
+              <SharedPage
+                folderInfo={
+                  <FolderInfo
+                    profileImage={profileImage}
+                    ownerName={ownerName}
+                    folderName={folderName}
+                  />
+                }
+                searchBar={<SearchBar />}
+                cardList={
+                  <CardList>
+                    {folders?.map((link) => (
+                      <ReadOnlyCard key={link?.id} {...link} />
+                    ))}
+                  </CardList>
+                }
+              />
```

Comment:
P2:
멘토링때 얘기 나왔던 것처럼 Route 컴포넌트에 넘겨주는 element를 따로 컴포넌트로 빼시면 더 깔끔할 것 같네요.
```suggestion
              <FolderPage /> // 해당 컴포넌트 내부에서 동일 로직 수행
```